src/App.js

```diff
@@ -0,0 +1,33 @@
+import './App.css';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Rockets from './Routes/rockets';
+import Missions from './Routes/missions';
+import Myprofile from './Routes/profile';
+
+import { getMissions } from './redux/mission/missionSlice';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, [dispatch]);
+
```

Comment:
- [ ] When the app loads, the missions should not be fetched from the API until a user navigates to the missions page. Only the rockets should be fetched. Kindly fix that.